code/game/turfs/simulated.dm

```diff
@@ -68,14 +68,6 @@
 						if (H.Slip(4, 5))
 							step(H, H.dir)
 							to_chat(H, "<span class='notice'>You tripped over your hair!</span>")
-
-		//Anything beyond that point will not fire if the mob isn't physically walking here
-		if(!M.on_foot()) //Checks lying, flying and locked.to
-			return ..()
-
-		//And anything beyond that point will not fire for slimes
-		if(isslime(M)) //Slimes just don't slip, end of story
```

Comment:
Is this covered by M.on_foot()?

Reply:
Nope. But slimes are never human and the functionality of that check got moved to here: https://github.com/vgstation-coders/vgstation13/blob/1b38ae662548099e03bb7b70331034ebd5483026/code/game/objects/effects/overlays.dm#L92

Now it's an early return at the end of the proc. It does nothing.

Reply:
Understandable